README.md

```diff
@@ -50,93 +88,98 @@ The following environment variables are available during site builds and when ru
 
 ## Development
 
-You'll need [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/) installed for development and testing.
-
-### Environment variables
+### Getting started
 
-To make setting environment variables easier for local development,
-create a new `.env` file based on the `.env.sample` and fill out the environment variables in it. Your `.env` file should **not be committed** to the repository
-because it will contain sensitive information. It is ignored by `.gitignore`.
+#### Requirements
+- [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/)
+- AWS S3 bucket name and associated credentials (key, secret, region)
+- A Github repository with a Federalist-compatible site
+- A Github Personal Access Token if building a private repository, see [creating a new personal token for your GitHub account](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) for more information.
 
+#### Clone the repository
 ```sh
-cp .env.sample .env
+  git clone git@github.com:18F/federalist-garden-build.git
+  cd federalist-garden-build
 ```
 
-For the AWS S3 values needed, you might find it helpful to
-spin up an [S3 service](https://cloud.gov/docs/services/s3/) in your [cloud.gov sandbox space](https://cloud.gov/overview/pricing/free-limited-sandbox/).
-
-For the `GITHUB_TOKEN`, create a new [personal token for your GitHub account](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) and use that.
+#### Create build arguments
+```sh
+  mkdir -p .local
+  cp .local.sample.json .local/my-build.json
```

Comment:
could it be helpful to user have either just have references to  "foo.json" or "my-build.json" ?
having both can make it appear they are different files

Reply:
oops, updating all to `my-build.json`